apps/frontend/src/app/sessions/select-play-level/client-page.tsx

```diff
@@ -94,24 +103,30 @@ export default function ClientSelectPlayLevelPage() {
   }
 
   return (
-    <div className="mx-4 flex h-dvh flex-col gap-y-4">
-      <BackNavigationBar pathName="/sessions" title="Select your level of play" />
+    <Container h="100dvh">
+      <VStack h="100%">
+        <BackNavigationBar pathName="/sessions" title="Select your level of play" />
 
-      {sortedSessions.map((session) => (
-        <div className="mb-4" key={session.id}>
-          <ExpandedSessionCard gameSession={session} />
-        </div>
-      ))}
+        <For each={sortedSessions}>
+          {(session) => (
+            <Box key={session.id} marginBottom="md">
+              <ExpandedSessionCard gameSession={session} />
+            </Box>
+          )}
+        </For>
```

Comment:
Nice use of `<For />`!!!
I think it could be like this tho.
```suggestion
        <For each={sortedSessions}>
          {(session) => <ExpandedSessionCard gameSession={session} key={session.id} />}
        </For>
```